run

```diff
@@ -34,7 +34,7 @@ boats = boat_ids.map do |ywid|
     ywid: ywid,
     href: href,
     title: boat_page.css("div.boat-title > h1").first.text.strip,
-    price: boat_page.css("div.boat-price").first.text.strip,
+    price: boat_page.css("div.boat-price").first.text.strip.scan(/\d/).join('').to_i,
```

Comment:
_non-blocking:_ wdyt about using `.tr('^0-9', '').to_i`? that would save us from converting back and forth between a string and an array.